src/Features/CSharp/Portable/CodeRefactorings/CSharpRefactoringHelpersService.cs

```diff
@@ -58,5 +61,20 @@ protected override IEnumerable<SyntaxNode> ExtractNodesSimple(SyntaxNode? node,
                 }
             }
         }
+
+        protected override SyntaxNode? ExtractSingleVariableFromFieldDeclaration(SyntaxNode root, int location)
```

Comment:
i'm not really sure this should be in the refactoring helper.  What about having the helper return the full fielddecl and have the consumer extract the variable?